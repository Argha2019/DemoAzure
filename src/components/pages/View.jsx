import React from 'react';
import ViewTemplate from "../templates/ViewTemplate/Template.jsx";
import '../../css/global.css';
import 'ag-grid-community/dist/styles/ag-grid.css';
import 'ag-grid-community/dist/styles/ag-theme-blue.css';
import ViewProfile from "./ViewProfile.jsx";


/**
 * @author Arghadeep Mandal
 * This file is responsible for making the header and footer template for role:"VIEW"
 * which is always rendered in every page. 
 */


export default class View extends React.Component {

    render() {
        return (
            <ViewTemplate>
                <ViewProfile />
            </ViewTemplate>
        );
    }
}
