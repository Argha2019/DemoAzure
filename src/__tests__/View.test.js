import React from 'react';
import { shallow } from 'enzyme';
// import App from '../App.jsx';
import toJson from 'enzyme-to-json';
import View from '../components/pages/View.jsx';

describe('View Component', () => {

    //generate snapshot
     it('should render snapshot', () => {
        const wrapper = shallow(<View />);
        expect(toJson(wrapper)).toMatchSnapshot();
      });
    })