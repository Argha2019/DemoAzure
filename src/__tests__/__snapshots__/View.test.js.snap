// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Component should render snapshot 1`] = `
<ViewTemplate>
  <ViewProfile />
</ViewTemplate>
`;
